hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -475,8 +485,8 @@ public void refreshTimeline() throws IOException {
         LOG.warn("Some records failed to be merged but forcing commit since commitOnErrors set. Errors/Total="
             + totalErrorRecords + "/" + totalRecords);
       }
-
-      boolean success = writeClient.commit(instantTime, writeStatusRDD, Option.of(checkpointCommitMetadata));
+      String commitActionType = CommitUtils.getCommitActionType(cfg.operation, HoodieTableType.valueOf(cfg.tableType));
```

Comment:
Since you folks did some investigation around this, Do you think we need to improve docs on 
WriteClient.startCommitWithTime(). There are two overloaded methods. may be clearly calling out when to call which one would benefit all.

Reply:
I meant to fix it in this PR itself. just one line java docs to each method. @codope : Can you coordinate w/ samrat and fix the docs.

Reply:
added the usage doc as per my understanding, please check if it needs improvement